importlib_metadata/__init__.py

```diff
@@ -975,14 +1150,13 @@ def entry_points(**params) -> Union[EntryPoints, SelectableGroups]:
     :return: EntryPoints or SelectableGroups for all installed packages.
     """
     norm_name = operator.attrgetter('_normalized_name')
-    unique = functools.partial(unique_everseen, key=norm_name)
```

Comment:
I think I mentioned it elsewhere, but one reason I wrote it this way was because I was thinking of exposing `unique` for other consumers to re-use (around other uses of `distributions()`).